client/lib/guides/trigger-guides-for-step.ts

```diff
@@ -7,8 +7,21 @@ import wpcom from 'calypso/lib/wp';
 const previousRequests = new Set< string >();
 
 // trigger guides on step movement, we don't care about failures or response
-export function triggerGuidesForStep( flowName: string, stepName?: string ) {
-	const key = flowName + ':' + ( stepName || '' );
+// modify the function to include siteSlug and siteId as optional
```

Comment:
Nitpick, but I'm not sure we need to add this as a comment to the code. The original comment should suffice.